index.js

```diff
@@ -160,8 +161,8 @@ exports.init = function (sbot, config) {
       log('tunnel:portal - received endpoint announcement from:'+this.id)
       endpoints[this.id] = sbot.peers[this.id][0]
     },
-    connect: function (opts, cb) {
-      if(!opts) return DuplexError('opts *must* be provided')
+    connect: function (opts) {
```

Comment:
The `cb` was not being used in this function

Reply:
Looks like you accidentally deleted the !opts check here?

Reply:
Nope, take a look below